.env.example

```
@@ -15,4 +15,10 @@ BETTER_AUTH_URL=
 BETTER_AUTH_SECRET=
 
 # Supabase
-DATABASE_URL=
\ No newline at end of file
+DATABASE_URL=
+
+# Resend
+RESEND_API_KEY=
+
+# Other
+NEXT_PUBLIC_APP_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)